### PR TITLE
Store new but for clothes

### PR DESCRIPTION
**Pull request by zkdan**

### CODE REVIEW

File: scripts.js [hunk 1]
```diff
@@ -1,84 +1,110 @@
...
+  },
+  {
+    title: 'HackerYou Hoodie',
+    price: 50.00,
+    stock: 4,
+  },
+]
 
-  // document ready function allows page to load before running any of our scripts 
-$(function(){
```
Comment (cwwyoung) [inline scripts.js:48]: hey Lee

File: scripts.js [hunk 2]
```diff
@@ -1,84 +1,110 @@
...
+    stock: 5,
+  },
+  {
+    title: 'Journey Tee',
+    url: 'images/journey.jpg',
+    price: 22.99,
+    stock: 6,
+  },
+  {
+    title: 'Skeleton Jean Jacket',
```
Comment (cwwyoung) [inline scripts.js:29]: this sounds like a really cool item!

File: scripts.js (hunk 1 above)
Comment (leetipton) [inline scripts.js:48]: Is this function bracketed wrong? Maybe...

**Review by leetipton — COMMENTED** [review]
Great job! Love code

**Pull request merged**